base/shell/explorer/traywnd.cpp

```diff
@@ -479,17 +479,15 @@ class CTrayWindow :
     DWORD WINAPI TrayPropertiesThread()
     {
         HWND hwnd;
-        RECT posRect;
 
-        m_StartButton.GetWindowRect(&posRect);
         hwnd = CreateWindowEx(0,
                               WC_STATIC,
                               NULL,
                               WS_OVERLAPPED | WS_DISABLED | WS_CLIPSIBLINGS | WS_BORDER | SS_LEFT,
-                              posRect.left,
-                              posRect.top,
-                              posRect.right - posRect.left,
-                              posRect.bottom - posRect.top,
```

Comment:
The idea of this code, iirc, was to display the dialog box on the bottom left (on left-to-right systems) or on the bottom right (on right-to-left systems) of the same screen as where the start menu button is. Of course that original code was buggy, but using the CW_USEDEFAULT things won't implement the expected position either.

Reply:
but we always have to move the window up !!

Reply:
As I said, it's because the current code is broken. Look at how the similar thing looks like on Windows.